### DISCUSSION TITLE
A how to manual might be a nice addition

### DISCUSSION THREAD

**Discussion by caring-coder** [ota4j-team/open-test-reporting#37] [General]
Hello, quite interested in this initiative. Trying out a few things.

I succeeded in generating a junit-platform-events-000....xml with junit platform, and I am now trying to convert it to the hierarchical format. It doesn't seem to work with the last release pushed to Sonatype (I seem to be able to execute it, but it doesn't do anything, and don't log, so I don't have a lot of leverage).

Am i supposed to build the tool myself from source ? If so, can we include that in the README.md or in a dedicated HOWTO.md ?

**Comment by peturingi** [comment]
Does it work using the latest version?

**Comment by caring-coder** [comment]
I'll check when free

**Comment by marcphilipp** [comment]
There are docs for the CLI tool now: https://github.com/ota4j-team/open-test-reporting#cli-tool

For JUnit, additional docs are available here: https://docs.junit.org/current/user-guide/#junit-platform-reporting-open-test-reporting
